ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -1,5 +1,75 @@
 [
   {
+    "id": "java-mysql",
+    "creator": "ide",
+    "name": "Java-MySQL",
+    "description": "Multi-machine environment with Default Java Stack and MySQL database",
+    "scope": "general",
+    "tags": [
+      "Java 1.8, Tomcat 8, MySQL 5.7"
+    ],
+    "components": [
```

Comment:
These are tags, smth that will show up in user dashboard. We use tags to match stacks and samples.

So, if I like versions of everything, we just do not have space in the stack box on UD.

Also, this will break all the filtering.